src/traccuracy/metrics/_divisions.py

```diff
@@ -32,45 +32,50 @@
 of the early division, by advancing along the graph to find nodes in the same frame
 as the late division daughters.
 """
+from __future__ import annotations
 
+from typing import TYPE_CHECKING
 
 from traccuracy._tracking_graph import NodeAttr
 from traccuracy.track_errors.divisions import _evaluate_division_events
 
 from ._base import Metric
 
+if TYPE_CHECKING:
+    from ._base import Matched
```

Comment:
This import isn't right, is it? It implies that `Matched` is in the metrics/_base.py

Reply:
Yeah you're right. I thought I had fixed that in the matcher PR, but maybe the change got lost in the merge.

Reply:
I think since we import `Matched` in `metrics/_base.py` it is exposed for subsequent import here. But it's super sketch